### ISSUE TITLE
Hide columns by default

### ISSUE THREAD

**Issue by tomzemp** [pepfar-datim/datim-mechanism-finder#40]
@benguaraldi asked that the following columns be hidden by default:

- Legacy ID
- Legacy Partner Name
- Legacy Partner ID
- ~Organization Type~
- Legacy Partner Organization Type ID

@mjenije asked that Organization type not be hidden

**Comment by tomzemp** [comment]
@benguaraldi: I can easily hide them, but in the sense that you cannot see them at all. Unfortunately, there doesn't seem (surprisingly) to be a property to collapse headers in react-table package, so if they're hidden they are gone from the table, rather than just collapsed
Is that okay?

**Comment by benguaraldi** [comment]
@tomzemp No, that's not a good idea.  Let's keep this ticket open and think about it later.  Thanks!
